spec/README.md

```diff
@@ -36,8 +36,8 @@ let(:provider) { course.provider }
 ## Overriding the creation of a dependent object
 
 ```
-let(:english_subject) { create :ucas_subject, :english }
-let(:course)          { create :course, ucas_subjects: [english_subject] }
+let(:english_subject) { create :subject, :english }
+let(:course)          { create :course, subjects: [english_subject] }
```

Comment:
probably should `level` it

Reply:
I think @defong means add a trait like `primary`.

Reply:
Going to do an additional PR to do this in the factories.